charts/substra-backend/values.yaml

```diff
@@ -835,6 +865,13 @@ postgresql:
       capabilities:
         drop:
         - ALL
+    resources:
+      requests:
+        cpu: "1000m"
+        memory: "2Gi"
+      limits:
+        cpu: "1000m"
+        memory: "4Gi"
```

Comment:
Seems pretty high but not too many CP ran in parallel.

<img width="1768" alt="image" src="https://github.com/Substra/substra-backend/assets/135698225/153e54c5-d091-49f4-9e63-a401c03f8a3b">

Reply:
Also, I think from past discussions with DBAs that PostGre is choosing different strategies to execute the queries depending on the amount of RAM available, so it's probably a bad idea for performance to restrict the RAM too much.
In fact, I was wondering if I should put an higher value, especially since the amount of data in the DB will only increase when the project is run in production.

Reply:
Ok to keep it as is for now, and monitor it in the coming days